new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -87,6 +88,40 @@ public static boolean isPodRunning(String namespace, String domainUid, String po
     return status;
   }
 
+  /**
+   * Checks if a pod is ready in a given namespace.
+   *
+   * @param namespace in which the pod is running
```

Comment:
namespace in which to check if the pod is ready

Reply:
fixed